br/Makefile

```diff
@@ -12,114 +12,72 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PROJECT=br
-GOPATH ?= $(shell go env GOPATH)
-P=8
+.PHONY: check test build clean 
+default: build
+all: check test build clean
```

Comment:
How about add a `dev` target ?
`make dev` will be required to be successful before submit PR.

`dev: check build test`

Reply:
maybe it's better to be contained in a [pre-commit](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) hook?

Reply:
otherwise there is always a way to skip running `make dev` before submitting a PR.